Source/DafnyCore/AST/Grammar/ParseErrors.cs

```diff
@@ -238,21 +322,24 @@ declared identifier.
     @"
 Because a mutable field does not have initializer, it must have a type (as in `var f: int`).
 `const` declarations may have initializers; if they do they do not need an explicit type.
-");
+", range => new List<DafnyAction> {
+    OneAction("insert ': bool'", range, range.PrintOriginal() + ": bool"),
+    OneAction("insert ': int'", range, range.PrintOriginal() + ": int")
+  });
 
     Add(ErrorId.p_no_init_for_var_field,
     @"
 Dafny does not allow field declarations to have initializers. They are initialized in constructors.
 Local variable declarations (which also begin with `var`) may have initializers.
-");
+"); // TODO - add action: remove := and expression
```

Comment:
Remove this todo, unless you have an idea how it is possible to implement at the parser level? The expression is not parsed yet.

Reply:
Parse the expression and then call the error -- implemented removing the := and expression

Reply:
Interesting idea! I guess that, if we parse the expression, I would prefer the code action to offer me to do the assignment in the constructor instead so that it does not waste my initialization. Could you add it as a comment?

Reply:
TODO comment added for future work